lp_bookdown/_output.yaml

```
@@ -0,0 +1,6 @@
+bookdown::word_document2:
+  toc: yes
+  toc_depth: 3
```

To keep the Table of contents shorter, I think we just want to show Chapter Headings/
```suggestion
  toc_depth: 1
```